backend/db/utils.py

```diff
@@ -24,6 +31,39 @@
 SCHEMA = CONFIG["schema"]
 
 
+# todo: move this somewhere else, possibly load.py or db_refresh.py
+def refresh_termhub_core_cset_derived_tables(con: Connection, schema: str):
```

Comment:
FYI: This function handles how these tables/views are refreshed:
        'cset_members_items',
        'members_items_summary',
        'cset_members_items_plus',
        'codeset_counts',
        'all_csets'